add rabbitmq to list of brokers (#3677)
knative/docs: docs/eventing/broker/README.md

For information about the Apache Kafka broker, see [link](./kafka-broker).

#### RabbitMQ broker

> The RabbitMQ Broker uses [RabbitMQ](https://www.rabbitmq.com/) for underlying implementation. For more details, refer to the [documentation](https://github.com/knative-sandbox/eventing-rabbitmq).

Reviewer: ```suggestion
The RabbitMQ Broker uses [RabbitMQ](https://www.rabbitmq.com/) for its underlying implementation. For more information, see the [RabbitMQ broker](https://github.com/knative-sandbox/eventing-rabbitmq) in GitHub.
```
